pkg/kubelet/cm/pod_container_manager_linux.go

```diff
@@ -197,6 +197,7 @@ func (m *podContainerManagerImpl) Destroy(podCgroup CgroupName) error {
 		ResourceParameters: &ResourceConfig{},
 	}
 	if err := m.cgroupManager.Destroy(containerConfig); err != nil {
+		klog.V(3).Infof("failed to delete cgroup paths for %v : %v", podCgroup, err)
```

Comment:
This should be at warning or error level.

Reply:
Actually I have a question, in the current code, the err is wrapped and returned.
Why is the new error not logged ?

Reply:
I would lean toward logging this as a warning.

Reply:
@tedyu ok, sounds good. changed it to a warning.